app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt

```diff
@@ -1444,7 +1445,8 @@ class StateFragmentTest {
 
     fun inject(stateFragmentTest: StateFragmentTest)
 
-    @IsOnRobolectric fun isOnRobolectric(): Boolean
```

Comment:
FYI this is technically conformant to our style guide: https://developer.android.com/kotlin/style-guide#annotations & ktlint normally seems amenable to it.